geo/src/algorithm/mod.rs

```diff
@@ -84,6 +82,10 @@ pub use densify::Densify;
 pub mod densify_haversine;
 pub use densify_haversine::DensifyHaversine;
 
+/// Densify geometry components on a spheroid;
```

Comment:
```suggestion
/// Densify geometry components on a geodesic.
```

nit: period, not semicolon.

And I don't know that it's better, but "geodesic"  is consistent with our other doc strings, so let's use it everywhere.